src/main/kotlin/step2/InputValidator.kt

```diff
@@ -0,0 +1,26 @@
+package step2
+
+class InputValidator {
+    companion object {
+        fun validate(input: String?) {
+            validateInputIsNullOrBlank(input)
+            validateIsArithmeticSymbol(input!!)
+        }
+
+        private fun validateInputIsNullOrBlank(input: String?) {
+            if (input.isNullOrBlank()) throw IllegalArgumentException()
+        }
```

Comment:
```suggestion
        require(input.isNullOrBlank())
```

kotlin에서는 `require` 함수를 통해서 `IllegalArgumentException`을 처리할 수 있습니다!

그리고 Exception 처리 시 오류에 대한 메시지도 같이 나타내줬으면 합니다 ☺️

참고
- [Kotlin 예외처리](https://seosh817.tistory.com/155)

Reply:
메시지 추가 하였습니다!